src/main/java/io/smartcat/ranger/core/parser/DataGenerator.java

```diff
@@ -63,13 +68,21 @@ public Builder(Map<String, Object> config, String outputValueName) {
          * @return Instance of {@link DataGenerator}.
          */
         public DataGenerator build() {
-            createProxies();
-            parseCompositeValue("", config);
-            return new DataGenerator(proxyValues.get(outputValueName));
+            if (values != null) {
+                createProxies();
+                parseCompositeValue("", values);
```

Comment:
`parseCompositeValue` is just here used as void, but on other place its return value is used.
`parseSimpleValue` is always used as return type, never as void.